src/main/java/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.impl;
+
+import db.Warehouse;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Map;
+import service.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String TOP_OF_REPORT = "fruit,quantity";
+    private static final String COMMA = ",";
```

Comment:
writer service should be a general class, it should only receive a context and a file name, it doesn't know about any format
lets create a separate service to create a report